web/error/handler.go

```diff
@@ -0,0 +1,57 @@
+package error
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"github.com/rs/zerolog"
+)
+
+func Handler(logger zerolog.Logger) echo.HTTPErrorHandler {
+	ll := logger.With().Str("middleware", "errorHandler").Logger()
+	return func(err error, c echo.Context) {
+		if c.Response().Committed {
+			return
+		}
+
+		ll.Err(err).
+			Str("requestID", c.Request().Header.Get(echo.HeaderXRequestID)).
+			Msg("request returned an error")
+
+		he, ok := err.(*echo.HTTPError)
+		if ok {
+			if he.Internal != nil {
+				if herr, ok := he.Internal.(*echo.HTTPError); ok {
+					he = herr
+				}
+			}
+		} else {
+			he = &echo.HTTPError{
+				Code:    http.StatusInternalServerError,
+				Message: http.StatusText(http.StatusInternalServerError),
+			}
+		}
+
+		// Issue #1426
+		code := he.Code
+		message := he.Message
+		if m, ok := he.Message.(string); ok {
+			if c.Echo().Debug {
+				message = echo.Map{"code": code, "message": m, "error": err.Error()}
+			} else {
+				message = echo.Map{"code": code, "message": m}
```

Comment:
Change 2 is here. The `"code": code, ` was added to both messages.

See original: https://github.com/labstack/echo/blob/82a964c657e26b68998393d3e7291f1a474447f8/echo.go#L414-L416